copter/source/docs/tuning-process-instructions.rst

```diff
@@ -96,16 +218,21 @@ Be aware that in this state the aircraft may be very slow to respond to large co
 Test Alt Hold
 -------------
 
+This test will allow to test the altitude controler and unsure the stability of your aircraft.
+
+1. Check :ref:`MOT_HOVER_LEARN <MOT_HOVER_LEARN>` is set to 2. This will allow the controler to learn by itself the correct hover value when flying.
+
 Take off in Stabilise and increase altitude to 5m. Switch to Alt_Hold and be ready to switch back. If the aircraft is hovering at a very low hover throttle value you may hear a reasonably fast oscillation in the motors.
 
-1. Check MOT_HOVER_LEARN is set to 2
 2. Ensure the aircraft has spent enough time in hover to let the hover throttle value converge to the correct value.
+
+# todo land ? or in flight ?
```

Comment:
2. Take off in Stabilise and increase altitude to 5m. Switch to Alt_Hold and be ready to switch back to Stabilize. If the aircraft is hovering at a very low hover throttle value you may hear a reasonably fast oscillation in the motors. Ensure the aircraft has spent at least 30 seconds in hover to let the hover throttle parameter converge to the correct value. Land and disarm the aircraft.

Reply:
ok. So we set the parameter on ground and not in flight !
after setting PSC_ACCZ_I and PSC_ACCZ_P, should we takeoff again to check the behavior ? or just continue to the next section is enough?

Reply:
Sorry, I don't understand what you are asking. The feedback there was to replace point 2 to address your todo.

Reply:
OH, the todo's are before what you are asking about! I thought they were after :)

They can do this in flight or on the ground but it is easier to do it on the ground if you are on your own. You can also use ch6 tuning for many parameters. If you are tuning with someone it is often easy enough to do it while in flight. I am happy to leave it up to the pilot.

My personal preference is to do parameter changes on the ground.